code/modules/client/preferences/species_features/lizard.dm

```diff
@@ -128,17 +128,6 @@
 	var/datum/species/species_type = preferences.read_preference(/datum/preference/choiced/species)
 	return initial(species_type.digitigrade_customization) == DIGITIGRADE_OPTIONAL
 
-
-/datum/preference/choiced/lizard_legs/is_accessible(datum/preferences/preferences)
-	. = ..()
-
-	if(!.)
-		return
-
-	var/datum/species/species_type = preferences.read_preference(/datum/preference/choiced/species)
-
-	return initial(species_type.digitigrade_customization) & DIGITIGRADE_OPTIONAL
```

Comment:
This one was ostensibly the more correct of the two, as it correctly checks `customization & DIGITIGRADE_OPTIONAL` instead of `customization == DIGITIGRADE_OPTIONAL`

Reply:
it's not a bitfield